src/App.js

```diff
@@ -70,35 +73,47 @@ class App {
 				// Success handler.
 				() => {
 					// Insert modified HTML.
-					$( '.mw-parser-output' ).html( self.api.getReplacementHtml() );
-					// Highlight when hover a user's contributions.
-					// @TODO This is just testing code and should be
-					// replaced by the proper behaviour.
-					$( '.mw-parser-output .editor-token' )
-						.on( 'mouseenter', function () {
-							const ids = self.api.getIdsFromElement( this );
-							// Activate all this user's contribution spans.
-							$( '.token-editor-' + ids.editorId ).addClass( 'active' );
-							// Information for popup.
-							// eslint-disable-next-line no-console
-							console.log( self.api.getTokenInfo( ids.tokenId ) );
-						} )
-						.on( 'mouseleave', function () {
-							// Deactivate all spans.
-							$( '.mw-parser-output .editor-token' ).removeClass( 'active' );
-						} );
-					self.widget.setState( 'ready' );
+					$( '.mw-parser-output' ).html( this.api.getReplacementHtml() );
+					this.attachContentListeners();
+					this.widget.setState( 'ready' );
 				},
 				// Error handler.
 				errorCode => {
-					self.widget.setState( 'err' );
-					self.widget.setErrorMessage( errorCode );
+					this.widget.setState( 'err' );
+					this.widget.setErrorMessage( errorCode );
 				}
 			);
 	}
 
 	/**
-	 * Repond to the close event that the widget emits.
+	 * Add listener to highlight attribution and show the RevisionPopupWidget.
+	 */
+	attachContentListeners() {
+		$( '.mw-parser-output .editor-token' )
+			.on( 'mouseenter', e => {
+				const ids = this.api.getIdsFromElement( e.target );
+				// Activate all this user's contribution spans.
+				$( '.token-editor-' + ids.editorId ).addClass( 'active' );
+			} )
+			.on( 'mouseleave', () => {
+				// Deactivate all spans.
+				$( '.mw-parser-output .editor-token' ).removeClass( 'active' );
+			} );
+
+		// Create RevisionPopupWidget instance.
+		const revisionPopup = new RevisionPopupWidget();
+		$( document.body ).append( revisionPopup.$element );
+
+		$( '.editor-token' ).on( 'click', e => {
+			const ids = this.api.getIdsFromElement( e.target );
+			revisionPopup.setContent( this.api.getTokenInfo( ids.tokenId ) );
```

Comment:
Arguably, you can group all of these into a single method in RevisionPopup... for example, something like this

```
RevisionPopup.prototype.processToken = function ( tokenInfo, $target ) {
     this.setContent( tokenInfo );
     this.setFloatableContainer( $target );
};
```

I can see pros and cons about having `toggle( true)` in or out of that method, but having it as a central method can centralize the general logic in case we want to add behavior or change it. Sounds a bit better architecturally, even though it's not crucial.